### PR TITLE
Add FireFly Graduation Proposal

### PR DESCRIPTION
**Pull request by nguyer**
This PR is the proposal to move Hyperledger FireFly to a Graduated Project. I am happy to discuss the proposal, or answer any questions the TOC or other community members may have here in the PR or on a call. Thanks!

### CODE REVIEW

**Review by petermetz — APPROVED** [review]
LGTM

File: HIPs/firefly_graduation.md
Comment (tkuhrt) [inline HIPs/firefly_graduation.md]: The grammar in the first sentence seems off. Maybe
```suggestion
Given that FireFly now meets or exceeds the [Incubation Exit Criteria](https://toc.hyperledger.org/governing-documents/project-incubation-exit.html) created by the Hyperledger TOC, we are proposing to move FireFly to Graduated Project status. The rest of this document describes in detail how the project meets or exceeds the Incubation Exit Criteria.
```

File: HIPs/firefly_graduation.md
Comment (tkuhrt) [inline HIPs/firefly_graduation.md:34]: With 10 maintainers coming from Kaleido, does this meet the `there is no single company or entity that is vital to the success of the project`? In other words, if Kaleido decided to stop supporting FireFly development, would the project continue to succeed? What is being done to move to a more diverse set of maintainers?

File: HIPs/firefly_graduation.md
Comment (tkuhrt) [inline HIPs/firefly_graduation.md:54]: Can you comment on the `various stages` and where you see each of these organizations at in the pipeline?

File: HIPs/firefly_graduation.md
Comment (tkuhrt) [inline HIPs/firefly_graduation.md]: Do the other repositories also have unit tests? If so, what are their code coverage?

File: HIPs/firefly_graduation.md
Comment (tkuhrt) [inline HIPs/firefly_graduation.md:82]: Is running the performance testing a one-off? Or is there data available for each of the releases on a given set of configurations? (Just curious...not a requirement for graduation.)

File: HIPs/firefly_graduation.md
Comment (tkuhrt) [inline HIPs/firefly_graduation.md:180]: Curious if you could point to the results of these independent tests.

File: HIPs/firefly_graduation.md
Comment (denyeart) [reply HIPs/firefly_graduation.md:34]: Specifically, the non-Kaleido maintainers appear to be active in the connectors, have they started to maintain the core Firefly repositories as well?

File: HIPs/firefly_graduation.md
Comment (nguyer) [reply HIPs/firefly_graduation.md]: Thanks. Yep that looks like a sentence I edited poorly.

File: HIPs/firefly_graduation.md
Comment (nguyer) [reply HIPs/firefly_graduation.md]: Yes, they do 🙂 I will list the other repositories here as well

File: HIPs/firefly_graduation.md
Comment (nguyer) [reply HIPs/firefly_graduation.md:34]: > What is being done to move to a more diverse set of maintainers?

This is a good question. I'm happy to elaborate on this more in the TOC call tomorrow.

> Specifically, the non-Kaleido maintainers appear to be active in the connectors, have they started to maintain the core Firefly repositories as well?

Here are @denisandreenko's commits in FireFly Core: https://github.com/hyperledger/firefly/commits?author=denisandreenko (Many of these were made earlier in the year but the branch was rebased before it got merged.)

I will also make the case that contributions to connectors should not be considered "second class" to contributions to the main FireFly repo. We call the main repo "FireFly Core" because it is the piece that all other pieces connect to. But the "Core" without connectors is useless. Additionally, the connectors contain the some of the most difficult, blockchain-specific code in the project.

File: HIPs/firefly_graduation.md
Comment (nguyer) [reply HIPs/firefly_graduation.md:54]: Yes! I'm planning on sharing more on this tomorrow during the call

File: HIPs/firefly_graduation.md
Comment (nguyer) [reply HIPs/firefly_graduation.md:82]: Archiving performance runs for previous releases is something we would like a better system for. Until now, when we're working on testing a release candidate we have bene creating a thread for the release in Discord. In that thread, we dump all of the settings that we're using for the performance test as well as results. It works in the moment, but it's not very accessible afterward. We're open to suggestions if other projects have found good solutions for this sort of thing.

File: HIPs/firefly_graduation.md
Comment (nguyer) [reply HIPs/firefly_graduation.md:180]: I would love to be able to, but this testing was done in the context of an NDA.

**Comment by tkuhrt** [comment]
**Approved at September 21, 2023 TOC meeting**

- Proposed: Bobbi
- Seconded: StephenC

Unanimous, 8/10 active TOC members 

- Tracy - For
- Arun - For
- Rama - For
- Bobbi - For
- StephenC - For
- Jim - For
- Peter - For
- Marcus - For

**Review by tkuhrt — APPROVED** [review]

**Pull request merged**
